packages/react-storage/src/components/StorageBrowser/validators/assertListCallerAccessGrantInput.ts

```diff
@@ -0,0 +1,9 @@
+export function assertListCallerAccessGrantInput(
```

Comment:
Think can just be `assertAccountId` since we may want to reuse it outside the context of calls to LCAG

Reply:
Done